sdk/keyvault/test-resources.json

```diff
@@ -1,223 +1,243 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
All the red part is due to vs reformatting the code, the useful changes contain the word secret

Reply:
I am a little worry about the re-formatting.  Do we know if it won't be re-formatted later from VSCode or some other tool?  
Do we have some format config for json?